### PR TITLE
LPS-144663

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi @tinatian ,
I have removed "portal.instance.protocol", and update it to "VerifyProperties.java".
Thanks for checking,
Jeff Wu

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -68,7 +68,7 @@
     #
     # Env: LIFERAY_PORTAL_PERIOD_INSTANCE_PERIOD_PROTOCOL
     #
-    portal.instance.protocol=
+
```
Comment (tinatian) [inline portal-impl/src/portal.properties:71]: @jeffwu0724 Please also remove the comments above and below. They are related to the removed property.

**Comment by tinatian** [comment]
@jeffwu0724 Please mentioned that the property is not used any more in the commit message, this is the reason we removed it.
Please see my comment above and resend.

Thanks.
Tina.

**Pull request closed without merging**